commons/connector/oura/oura_sleep.avsc

```diff
@@ -0,0 +1,45 @@
+    {"name": "deepSleepDuration", "type": ["null", "int"], "doc": "Duration spent in deep sleep in seconds.", "default": null},    
+    {"name": "efficiency", "type": ["null", "int"], "doc": "Sleep efficiency rating in range [1, 100].", "default": null},    
+    {"name": "latency", "type": ["null", "int"], "doc": "Sleep latency in seconds. This is the time it took for the user to fall asleep after going to bed.", "default": null},    
+    {"name": "lightSleepDuration", "type": ["null", "int"], "doc": "Duration spent in light sleep in seconds.", "default": null},    
+    {"name": "lowBatteryAlert", "type": ["null", "boolean"], "doc": "Flag indicating if a low battery alert occurred.", "default": null},    
+    {"name": "lowestHeartRate", "type": ["null", "int"], "doc": "Lowest heart rate during sleep.", "default": null},    
+    {"name": "movementThirtySec", "type": ["null", "string"], "doc": "The 30-second movement classification for the period where every character corresponds to: '1' = no motion, '2' = restless, '3' = tossing and turning '4' = active.", "default": null},    
+    {"name": "period", "type": ["null", "int"], "doc": "ECore sleep period identifier.", "default": null},    
+    {"name": "readinessContributorActivityBalance", "type": ["null", "int"], "doc": "Contribution of cumulative activity balance to the readiness score in range [1, 100].", "default": null},    
+    {"name": "readinessContributorBodyTemperature", "type": ["null", "int"], "doc": "Contribution of cumulative activity balance to the readiness score in range [1, 100].", "default": null},    
+    {"name": "readinessContributorHrvBalance", "type": ["null", "int"], "doc": "Contribution of cumulative activity balance to the readiness score in range [1, 100].", "default": null},    
+    {"name": "readinessContributorPreviousDayActivity", "type": ["null", "int"], "doc": "Contribution of cumulative activity balance to the readiness score in range [1, 100].", "default": null},    
+    {"name": "readinessContributorPreviousNight", "type": ["null", "int"], "doc": "Contribution of cumulative activity balance to the readiness score in range [1, 100].", "default": null},    
+    {"name": "readinessContributorRecoveryIndex", "type": ["null", "int"], "doc": "Contribution of cumulative activity balance to the readiness score in range [1, 100].", "default": null},    
+    {"name": "readinessContributorRestingHeartRate", "type": ["null", "int"], "doc": "Contribution of cumulative activity balance to the readiness score in range [1, 100].", "default": null},    
+    {"name": "readinessContributorSleepBalance", "type": ["null", "int"], "doc": "Contribution of cumulative activity balance to the readiness score in range [1, 100].", "default": null},    
+    {"name": "readinessScore", "type": ["null", "int"], "doc": "Readiness score for this sleep.", "default": null},    
```

Comment:
@yatharthranjan There's readiness data here during sleep, but I also have a separate `oura_readiness` schema. Should I just use that separate schema and remove from here?